src/platform/forms/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -90,7 +90,29 @@ class RoutedSavableApp extends React.Component {
     ) {
       newProps.router.replace(newProps.returnUrl);
     } else if (status === LOAD_STATUSES.success) {
-      newProps.router.push(newProps.returnUrl);
+      // TODO: Remove this when 526 v1 is depreciated
```

Comment:
Would it cause much trouble to make an `onFormLoaded` hook in the form config that was called here and move all of this logic out of RoutedSavableApp and into the form specific code? Even if this is really short term, it's still 526 specific code in our site wide SiP components.

Reply:
I dig it! Good thinking. :raised_hands: